public class lesson_15_10 {
    //Выбрасывание исключений

    //Оператор throw

    //Исключение, пока оно не выброшено в систему, или когда оно уже перехвачено блоком catch — это в принципе просто объект класса,
    //унаследованного от Exception (вернее, от Throwable). Никакими магическими особенностями сам объект-исключение не обладает.
    //Вся логика работы исключений — это просто особый сценарий поведения Java-машины, когда в ее среду выполнения вброшено исключение.
    //Вы всегда можете снова вбросить Java-машине исключение, которое только что перехватили. Для этого нужно использовать оператор throw:
    //throw исключение;


}
